Resources/Private/TypoScript/NodeTypes/NewsList.ts2

```diff
@@ -35,7 +35,9 @@ prototype(JohannesSteu.Neos.News:NewsList) < prototype(Flowpack.Listable:ListNod
         collection.@process.respectImportantNews = ${q(value).count() > 0 && q(value).removeImportantNews(respectImportant).get()}
 
         # apply newsfilter if set
+        collection.@process.applyCategoryFilter = ${request.arguments.category != NULL ? q(value).filterByCategory(request.arguments.category).get() : value}
```

Comment:
please prefix the argument with `newsfilter` as `category` is pretty much global and could also be used for something different